x-pack/plugins/observability/server/services/slo/transform_generators/__snapshots__/kql_custom.test.ts.snap

```diff
@@ -535,42 +356,12 @@ Object {
         },
         "type": "keyword",
       },
-      "slo.name": Object {
-        "script": Object {
-          "source": "emit('irrelevant')",
-        },
-        "type": "keyword",
-      },
-      "slo.objective.target": Object {
-        "script": Object {
-          "source": "emit(0.999)",
-        },
-        "type": "double",
-      },
```

Comment:
What's the purpose of remaining `runtime_mappings` in the source index?
i am still seeing `slo.id` `slo.revision` and `slo.instanceId` in runtime_mappings.

Reply:
So we are injecting this field through runtime_mapping, so we can groupBy on them in the aggregation. The source data does not have the slo.id, revision etc, so we have to inject them in order to group on them :) We asked the transform team to provide a more efficient way of doing this, e.g. "static fields"